web/src/components/Diagram/components/DAG/ControlNavigator.tsx

```diff
@@ -1,22 +1,26 @@
 import {LeftOutlined, RightOutlined} from '@ant-design/icons';
+
 import {useSpan} from 'providers/Span/Span.provider';
 import * as S from './DAG.styled';
 import useControls from './hooks/useControls';
 import {useListenToArrowKeysDown} from './useListenToArrowKeysDown';
 
-const AffectedSpanControls = () => {
-  const {affectedSpans, onSelectSpan} = useSpan();
-  const {handleNextSpan, handlePrevSpan, indexOfFocused} = useControls({onSelectSpan});
+const ControlNavigator = () => {
+  const {affectedSpans} = useSpan();
+  const {handleNextSpan, handlePrevSpan, indexOfFocused} = useControls();
   useListenToArrowKeysDown(affectedSpans, handleNextSpan, handlePrevSpan);
+
+  if (!affectedSpans.length) return <div />;
```

Comment:
What if a selector doesn't match any spans so it returns an empty affected spans array?